### PR TITLE
feat: Integrating Sitemap using next-sitemap package

### PR DESCRIPTION
**Pull request by leonboripgs**

### CODE REVIEW

**Review by narthur — APPROVED** [review]
Looks good!

Can you help me understand which pages do and don't get included and why? Thanks!

https://audioverse-next-git-fork-leonboripgs-sitemap-audioverse.vercel.app/sitemap-0.xml

File: next-sitemap.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/** @type {import('next-sitemap').IConfig} */
+
+module.exports = {
+	siteUrl: process.env.SITE_URL || 'https://audioverse.org',
+	generateRobotsTxt: true,
```
Comment (mattleff) [inline next-sitemap.js:5]: This doesn't appear to be working since the `public/robots.txt` exists.

**Review by mattleff — CHANGES_REQUESTED** [review]
It appears this only includes the pages that are being pre-rendered. I don't know if that's a problem or not...

**Comment by marcomiduri** [comment]
@mattleff @narthur Would you please check the PR again?

**Comment by marcomiduri** [comment]
> Looks good!
> 
> Can you help me understand which pages do and don't get included and why? Thanks!
> 
> https://audioverse-next-git-fork-leonboripgs-sitemap-audioverse.vercel.app/sitemap-0.xml

@narthur Since we didn't set the configuration like `exclude`, all pages are added to XML file because `next-sitemap` command checks all generated HTML files and add them to sitemap. You can review this link https://github.com/iamvishnusankar/next-sitemap#configuration-options

**Review by narthur — APPROVED** [review]
Changes look good. After talking with @mattleff I'm fine with the sitemap not including pages that aren't created at build time.

**Comment by marcomiduri** [comment]
> Changes look good. After talking with @mattleff I'm fine with the sitemap not including pages that aren't created at build time.

@narthur Thanks for your review, do you know why the build is failed on github action?

**Review by mattleff — APPROVED** [review]

**Pull request merged**